### DISCUSSION TITLE
Welcome to Steamodded Discussions!

### DISCUSSION THREAD

**Discussion by Steamopollys** [Steamodded/smods#352] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stann-co** [comment]
i have only done very little modding. Adding 2 or so custom cards with help. 
![Aseprite_dQwqJExNvh](https://github.com/user-attachments/assets/b6e4415c-f78d-47de-a3aa-7ebbfc7e8d57)
this is my most recent : )

**Reply by Chapfallen** [reply]
Nice card. What does it do?

**Comment by MathIsFun0** [comment]
m

**Reply by Aurelius7309** [reply]
m

**Reply by WilsontheWolf** [reply]
m

**Reply by cg-223** [reply]
m

**Reply by DigitalDetective47** [reply]
m

**Reply by GauntletGames-2086** [reply]
m

**Reply by AveryKH** [reply]
m

**Reply by baltrofan88** [reply]
m

**Reply by moxie-coder** [reply]
m
